__tests__/app.test.js

```diff
@@ -251,6 +251,7 @@ describe("GET /api/articles/:article_id", () => {
                 expect(article.article_img_url).toBe(
                     "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
                 )
+                expect(article.comment_count).toBe(11)
```

Comment:
Rather than adding this line in here, we should put it as a separate test. The previous test should still pass if you've used a flexible matcher 👍